nvflare/fuel/f3/drivers/aio_grpc_driver.py

```diff
@@ -39,7 +39,7 @@
 GRPC_DEFAULT_OPTIONS = [
     ("grpc.max_send_message_length", MAX_FRAME_SIZE),
     ("grpc.max_receive_message_length", MAX_FRAME_SIZE),
-    ("grpc.keepalive_time_ms", 120000),
+    ("grpc.keepalive_time_ms", 300000),
```

Comment:
What should be a suitable number for this?
@chesterxgchen @IsaacYangSLA Do you remember the azure timeout stuff, what is that magic number

Reply:
Changed to leave the value at 120000 (2 min).